.storybook/preview-head.html

```diff
@@ -16,6 +16,186 @@
     background: url('data:image/svg+xml;utf8,<svg height="6" width="1" xmlns="http://www.w3.org/2000/svg"><rect width="1" height="0.5" style="fill:rgba(120, 240, 215, 0.82)" /></svg>')
       repeat top left;
   }
+
```

Comment:
Are all of these duplicated styles needed? We also load the root css file in file in preview.js. It seems to do the job.